Formula/b/bigquery-emulator.rb

```diff
@@ -0,0 +1,34 @@
+class BigqueryEmulator < Formula
+  desc "Emulate a GCP BigQuery server on your local machine"
+  homepage "https://github.com/goccy/bigquery-emulator"
+  url "https://github.com/goccy/bigquery-emulator/archive/refs/tags/v0.6.1.tar.gz"
+  sha256 "4f8c037d03cd23d2a44d74460b5399213e0efeb33d6cade25bfce25499c4699a"
+  license "MIT"
+  head "https://github.com/goccy/bigquery-emulator.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "llvm" => :build
+
+  uses_from_macos "netcat" => :test
+
+  def install
+    ENV["CGO_ENABLED"] = "1"
+    ENV.clang
```

Comment:
Let's use `fails_with` to make `brew` use `clang` on Linux instead. You can `grep` for many examples in this repo.

Reply:
I've pushed this on latest commit, thanks.